draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -51,11 +52,11 @@ $large-width: 450px;
 
 %box {
   background: white;
-  border: $kz-border-solid-border-width $kz-border-solid-border-style
-    $kz-color-wisteria-200;
+  border: $kz-var-border-solid-border-width $kz-var-border-solid-border-style
+    $kz-var-color-ash;
```

Comment:
Should we be using `var($kz-var-id-color-ash, $kz-color-wisteria-200)` here for the fallback on Zen?

Reply:
Oh damn missed it thank you!